start-flask.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash -e
 
+# set REVISION env var from commit_id.txt or default
+export COMMIT_ID=$(cat commit_id.txt 2> /dev/null)
```

Comment:
It's much better to inject the REVISION arg at build / run time via env vars, avoid using a file to store this data which then creates the env var...it's easier to just inject an env var directly. 

I think we should use the build-args ci / cid branch you are testing out https://github.com/zooniverse/interventions-gateway/pull/62#pullrequestreview-930049040

Reply:
Doesn't seem like much of a performance hit if it's only reading the file more than once per app start. Injecting the env var may be preferable for other reasons, though, and if the app doesn't use the file anywhere else and since that's what this repo was doing originally, I've reverted to that method via the new optional argument to the reusable workflow.